{"priority": "major", "kind": "bug", "repository": {"links": {"self": {"href": "data/repositories/osrf/vrx.json"}, "html": {"href": "#!/osrf/vrx"}, "avatar": {"href": "data/bytebucket.org/ravatar/{e278ca38-7edb-4e62-b785-46dff5617d98}ts=2274605"}}, "type": "repository", "name": "vrx", "full_name": "osrf/vrx", "uuid": "{e278ca38-7edb-4e62-b785-46dff5617d98}"}, "links": {"attachments": {"href": "data/repositories/osrf/vrx/issues/62/attachments_page=1.json"}, "self": {"href": "data/repositories/osrf/vrx/issues/62.json"}, "watch": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/vrx/issues/62/watch"}, "comments": {"href": "data/repositories/osrf/vrx/issues/62/comments_page=1.json"}, "html": {"href": "#!/osrf/vrx/issues/62/adding-sensors-custom-urdf-not-working"}, "vote": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/vrx/issues/62/vote"}}, "reporter": {"display_name": "Matthew", "uuid": "{4fab3323-7162-480a-9d31-3ffa84c268c0}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B4fab3323-7162-480a-9d31-3ffa84c268c0%7D"}, "html": {"href": "https://bitbucket.org/%7B4fab3323-7162-480a-9d31-3ffa84c268c0%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/5c63d9db9854722ed6777d28/fe766f71-ac94-4753-9b6d-5bac427edda4/128"}}, "nickname": "MattBowers", "type": "user", "account_id": "5c63d9db9854722ed6777d28"}, "title": "Adding Sensors: Custom .urdf not working with updated scripts", "component": null, "votes": 0, "watches": 1, "content": {"raw": "Hello, my name is Matt Bowers. I'm experiencing a problem with getting a custom .urdf working and the problem may be stemming from the newer scripts in the tutorial since my .urdf was developed. Specifically, the error occurs when trying to spawn the wamv. I will try to attach a picture illustrating the problem.", "markup": "markdown", "html": "<p>Hello, my name is Matt Bowers. I'm experiencing a problem with getting a custom .urdf working and the problem may be stemming from the newer scripts in the tutorial since my .urdf was developed. Specifically, the error occurs when trying to spawn the wamv. I will try to attach a picture illustrating the problem.</p>", "type": "rendered"}, "assignee": null, "state": "resolved", "version": null, "edited_on": null, "created_on": "2019-02-20T03:53:16.830234+00:00", "milestone": null, "updated_on": "2019-03-01T19:50:57.335312+00:00", "type": "issue", "id": 62}